src/main/webapp/index.html

```diff
@@ -20,6 +20,7 @@
     <script src="https://kit.fontawesome.com/de2fa200e7.js" crossorigin="anonymous"></script>
     </link>
     <script src="script.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=MAP_API_KEY"></script>
```

Comment:
won't we need an api key here?

Reply:
Yeah but it’s not safe to upload it to github. If you want to test locally you could just put your own MAP_API Key or I could send mine.

Reply:
ah ok i see i see. I think charles put on though, so we should be wary of that